src/main/scala/org/broadinstitute/dsde/vault/services/analysis/UpdateServiceHandler.scala

```diff
@@ -94,7 +94,7 @@ case class UpdateServiceHandler(requestContext: RequestContext, dmService: Actor
         case true =>
           if (fileCount == bossObjects.size) {
             log.debug("'X-Force-Location' Analysis update complete")
-            requestContext.complete(new Analysis(dmId.get, analysis.input, analysis.metadata, analysis.files))
+            requestContext.complete(new Analysis(dmId.get, analysis.input, analysis.metadata, Option(providedFiles)))
```

Comment:
Nope!  By testing, it's the DM ID in both cases.  So we do need to override this in all cases.